lib/planner.go

```diff
@@ -28,6 +28,9 @@ import (
 
 //CreatePlan creates the plan from all planners
 func CreatePlan(ctx context.Context, inputPath, outputPath string, customizationsPath, transformerSelector, prjName string) plantypes.Plan {
+	if inputPath == "" && customizationsPath == "" {
```

Comment:
Why should it fail when both of these are empty? Even internal transformers can be invoked by default, isn't it

Reply:
Ah, you are right!